.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+
+Given that …,
```

Comment:
I'm not sure that tests way of doing that stuff looks helpful here. I think it better just to write what are the steps that need to be done for getting the incorrect behavior (or what steps have done, if it hard to reproduce)